js/apps/viewer/src/app/snapshot/snapshot.tsx

```diff
@@ -404,13 +405,13 @@ function setMimeType(data: string, format?: ImageFormat) {
     return data;
 }
 
-function defaultSize(): {
+function defaultSize(currentWidth: Breakpoint): {
     width: number;
     height: number;
 } {
     const [width, height] = [
         window.innerWidth,
-        window.innerHeight
+        window.innerHeight - (isWidthDown('xs', currentWidth) ? 64 : 56)
```

Comment:
magic number looks wrong. why not using the current viewport of Brayns?

Reply:
I made some consts now. We cannot use the viewport from Brayns because other clients could change that and the snapshot default size should be the size of the browser's viewport that the snapshot is taken in.